engine/query-constants/src/main/java/io/deephaven/util/QueryConstants.java

```diff
@@ -187,7 +187,13 @@ private QueryConstants() {}
     /**
      * Minimum finite value of type float.
      */
-    public static final float MIN_FINITE_FLOAT = Math.nextUp(-Float.MAX_VALUE);
+    /*
+     * Implemented as a constant, so that it can be translated correctly to JS, without needing a proper Math.nextUp()
+     * or nextAfter() implementation that works in JavaScript. Value is generated through running
+     * "Float.toHexString(Math.nextUp(-Float.MAX_VALUE))" in jshell, and tests will validate that this matches the
+     * expected expression.
```

Comment:
I know it's not actually JavaDoc, but I could see it being an `@implNote` at some point.
```suggestion
     * Implemented as a constant so that it can be translated correctly to JS without needing a proper
     * {@code Math.nextUp()} or {@code Math.nextAfter()} implementation that works in JavaScript. Value is generated by
     * running {@code Float.toHexString(Math.nextUp(-Float.MAX_VALUE))} in jshell, and tests validate that this matches
     * the expected expression.
```

Reply:
Note that this isn't a javadoc comment, but just a plain block comment, so I'm not sure tags do us any good.

Reply:
>I know it's not actually JavaDoc, but I could see it being an @implNote at some point.